test/controllers/billing-accounts.controller.test.js

```diff
@@ -46,6 +47,34 @@ describe('Billing Accounts controller', () => {
     Sinon.restore()
   })
 
+  describe('/billing-accounts/{billingAccountId}', () => {
+    describe('GET', () => {
+      beforeEach(async () => {
```

Comment:
This shouldn't need to be `async`
```suggestion
      beforeEach(() => {
```